inc/nav-moh.php

```diff
@@ -1,39 +1,62 @@
 <?php
 /**
- * This partial contains site navigation for the music oral history resource.
+ * The template for displaying the child navigation bar.
  *
- * @link https://github.com/MITLibraries/music-oral-history
- *
- * @package WordPress
- * @subpackage music-oral-history theme
- * @since 1.1.11
+ * @package MIT_Libraries_Child
+ * @since 2.0.0
  */
 
+  $siteName = get_bloginfo( 'name' );
+  $noChildNav = array( 'MIT Libraries News', 'Document Services' );
```

Comment:
I don't think we need this Doc services line.

Reply:
ok